lib/systems/default.nix

```diff
@@ -183,6 +183,26 @@ let
          || isWasm                                                     # WASM
         ) && !isStatic;
 
+      # Controls whether libunwind is enabled.
+      hasStackUnwinding = !(final.isWasm || final.isFreeBSD || final.isDarwin || final.isNone);
```

Comment:
I think we might need to split this, because these cases are slightly different:
1. On FreeBSD and Darwin there _is_ unwinding, but we want to use the system `libunwind`.
2. On WASM and bare metal we don't want unwinding at all.